<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="URMILA-PC" name="testRunner.MainRunner" tests="3" failures="2" timestamp="4 Aug 2020 06:31:33 GMT" time="144.351" errors="0">
  <testcase name="runScenario" time="56.370" classname="testRunner.MainRunner"/>
  <testcase name="runScenario" time="60.409" classname="testRunner.MainRunner">
    <failure type="java.lang.AssertionError" message="expected:&lt;Your store. Login&gt; but was:&lt;Dashboard / nopCommerce administration&gt;">
      <![CDATA[java.lang.AssertionError: expected:<Your store. Login> but was:<Dashboard / nopCommerce administration>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at stepDefinations.LoginStepDefination.user_should_be_taken_to_the_successful_login_page(LoginStepDefination.java:63)
	at ✽.User should be taken to the successful login page(file:///E:/Automation/Automation%20Workspace/WS_finsys/Cucumber_Framework/Features/login.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="27.572" classname="testRunner.MainRunner">
    <failure type="java.lang.AssertionError" message="expected:&lt;Your store. Login&gt; but was:&lt;Dashboard / nopCommerce administration&gt;">
      <![CDATA[java.lang.AssertionError: expected:<Your store. Login> but was:<Dashboard / nopCommerce administration>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at stepDefinations.LoginStepDefination.user_should_be_taken_to_the_successful_login_page(LoginStepDefination.java:63)
	at ✽.User should be taken to the successful login page(file:///E:/Automation/Automation%20Workspace/WS_finsys/Cucumber_Framework/Features/login.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- testRunner.MainRunner -->
